pyproject.toml

```diff
@@ -526,9 +526,9 @@ convention = "numpy"
         type_incorrect_long = "The changelog file you added is not one of the allowed types. Please use one of the types described in the changelog [README](https://github.com/astropy/astropy/blob/main/docs/changes/README.rst)"
         number_incorrect_long = "The number in the changelog file you added does not match the number of this pull request. Please rename the file."
 
-    [tool.gilesbot.circleci_artifacts.py39-test-image-mpl311]
-        url = ".tmp/py39-test-image-mpl311/results/fig_comparison.html"
-        message = "Click details to see the figure test comparisons, for py39-test-image-mpl311."
+    [tool.gilesbot.circleci_artifacts.py39-test-image-mpl322]
```

Comment:
Shouldn't the `mpl322` be `mpl33x` now with the matplotlib minversion bump?

Reply:
Might need to generate new truths for https://github.com/astropy/astropy-figure-tests too.

Reply:
Maybe @astrofrog or @Cadair can confirm of such procedures when we bump matplotlib minversion.

I think the CircleCI config also needs updating.

Reply:
Yes, help with that would be appreciated!

Reply:
@pllim, I added the Changelog you suggested.

Reply:
> I think bumping matplotlib needs more work.

@pllim , other than this thread, what else about bumping matplotlib needs doing?